src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -75,6 +75,274 @@ curie_map:
   GC_ID: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/GC_ID/"
   SNOMEDCT_2010_1_31: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/SNOMEDCT_2010_1_31/"
 
+extended_prefix_map:
+  - prefix: MONDO
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/MONDO_
+    uri_prefix_synonyms: []
+  - prefix: UMLS
+    prefix_synonyms: []
+    uri_prefix: http://linkedlifedata.com/resource/umls/id/
+    uri_prefix_synonyms: []
+  - prefix: NCIT
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/NCIT_
+    uri_prefix_synonyms: []
+  - prefix: DOID
+    prefix_synonyms: []
+    uri_prefix: http://purl.obolibrary.org/obo/DOID_
+    uri_prefix_synonyms: []
+  - prefix: EFO
+    prefix_synonyms: []
+    uri_prefix: http://www.ebi.ac.uk/efo/EFO_
+    uri_prefix_synonyms: []
+  - prefix: HGNC
```

Comment:
## URI prefix synonyms
For several namespaces/prefixes, such as HGNC, I've combined what were previously several prefixes in `curie_map` (e.g. `HGNC` and `HGNC__2`, etc) into 1 EPM entry, e.g.:

```yaml
    uri_prefix_synonyms:
      - 'https://identifiers.org/hgnc:'
      - 'https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:'
````